extensions/objects/models/objects.py

```diff
@@ -1146,3 +1146,25 @@ class PositionOfTerms(BaseObject):
         'type': 'unicode',
         'choices': ['lhs', 'rhs', 'both', 'irrelevant']
     }
+
+
+class CustomOskLetters(BaseObject):
+    """Class fot custom OSK letters. These are the letters that will be
+    displayed to the learner for AlgebraicExpressionInput and MathEquationInput
+    interactions when the on-screen keyboard is being used. This includes roman
```

Comment:
Do you mean 'roman' or 'latin'? Or are they the same :P?

Reply:
Apparently they're the same ([ref](https://en.wikipedia.org/wiki/Latin_alphabet)), but just to be consistent, I've changed this to Latin, since I've used Latin everywhere else.